src/quarterback/api/methods.ts

```diff
@@ -29,8 +29,16 @@ const DEFAULT_HEADERS = {
   'Content-Type': 'application/json',
 }
 
+let debouncedAuth: ReturnType<typeof setTimeout> | undefined
+
 function getAuthHeader() {
   const jwt = useAuthStore.getState().jwt
+  if (!jwt && !debouncedAuth) {
+    useAuthStore.getState().authenticate()
```

Comment:
Don't you want to re assign (get) the JWT value after this step? or maybe it's fine as the second attempt to getAuthHeader should work.

Reply:
Yeah the request will fail but I didnt want to complicate it too much. After all it should work in the first place 😅